source/documentdb-support.txt

```diff
@@ -69,6 +69,16 @@ available in a limited capacity or are not supported at all:
   ``$toObjectId``, ``$toString``, ``$isNumber``, ``$rand``, ``$sampleRate``, ``$getField``
 - **Aggregation Stages**: ``$collStats``, ``$facet``, ``$bucket``, ``$bucketAuto``,
   ``$sortByCount``, ``$listLocalSessions``, ``$listSessions``, ``$graphLookup``
+  
+Amazon DocumentDB also does not leverage indexes when you perform
+queries that contain any of the following operators:
```

Comment:
S: ticket mentions this too

```suggestion
case-sensitive queries or queries that contain any of the following operators:
```

Reply:
this is already listed as a non compatible feature in the preceding list!